tests/performance/config/perf-driver/environments/target-cluster-custom.yml

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------- #
+# Configuration Fragment : Custom DC/OS Cluster Target        #
+# ----------------------------------------------------------- #
+# This fragment instructs the driver to use the marathon      #
+# exposed from an enterprise cluster. In contrast to other    #
+# cluster targets it requires the auth token to be given by   #
+# the user.                                                   #
+# ----------------------------------------------------------- #
+
+# Global test configuration
+# ===========================
+config:
```

Comment:
Is this already interpreted by the per driver? Where is its implementation?

Reply:
Yes, this is part of the [standard driver configuration](http://dcos-performance-test-driver.readthedocs.io/en/latest/config/Readme.html). 

The driver itself is supposed to be available in the user's environment before running the manual script.